include/udx.h

```diff
@@ -267,7 +275,7 @@ struct udx_socket_send_s {
 };
 
 struct udx_stream_write_s {
-  uint32_t packets;
+  int64_t bytes;
```

Comment:
Any reason for this being signed? Why not `size_t`?

Reply:
changed